tutorials/algorithms/11_gradients_framework.ipynb

```diff
@@ -0,0 +1,741 @@
+{
```

Comment:
<p>I think some references on gradients and the QFI would be good here.</p><p>https://quantum-journal.org/papers/q-2021-09-09-539/</p><p>https://quantum-journal.org/papers/q-2020-05-25-269/</p><p>https://journals.aps.org/pra/abstract/10.1103/PhysRevA.103.012405</p><p>https://journals.aps.org/pra/abstract/10.1103/PhysRevA.99.032331</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1463/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>I think a single reference on gradients will suffice. Have a reference for QFI, QGT, and quantum gradients each. </p>